execute/transport_internal_test.go

```diff
@@ -1,7 +1,14 @@
 package execute
 
-import "github.com/influxdata/flux"
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute/table"
+)
 
 func NewProcessMsg(tbl flux.Table) ProcessMsg {
 	return &processMsg{table: tbl}
 }
+
+func NewProcessChunkMsg(chunk table.Chunk) ProcessChunkMsg {
+	return &processChunkMsg{chunk: chunk}
+}
```

Comment:
It's to avoid exposing these types to the public Go API while allowing them to be used in tests. It's a weird trick to black box test while accessing "internal" code.

When the Go compiler is compiling tests, it will include any files that have `_test.go`. At the same time, it allows there to be a test package that has a different name than the primary package and to compile them together. So this file only gets compiled in when tests are compiled but otherwise pretends it is part of the `execute` package instead of the `execute_test` package so it gets access to the internal functions defined in that package.